packages/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+### Install dependencies
```

Comment:
How is this relating to contributing?

Reply:
This is currently under a `Development` heading in the readme. I should head it under the same. https://github.com/readmeio/metrics-sdks/tree/main/packages/python#development